### PR TITLE
(CDAP-2366) call destroy() appropriately

### PR DESCRIPTION
**Pull request by chtyim**
- Introduced Destroyable to unify destroy call.

### CODE REVIEW

**Comment by gokulavasan** [comment]
LGTM :+1:

File: cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/Destroyables.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+package co.cask.cdap.templates.etl.api;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Helper class to deal with {@link Destroyable}.
+ */
+public final class Destroyables {
```
Comment (gokulavasan) [inline cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/Destroyables.java]: Does this need to be in etl-api?

**Comment by gokulavasan** [comment]
One comment. Rest LGTM :+1:

File: cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/Destroyables.java (hunk 1 above)
Comment (chtyim) [inline cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/Destroyables.java]: Moved

**Pull request merged**